### DISCUSSION TITLE
Cascading States css properties

### DISCUSSION THREAD

**Discussion by su-angel** [tailwindlabs/tailwindcss#14731] [Ideas]
Hello, hope you are fine 
I want to know if there is a possibility for cascading tailwind's properties on state declaration.
this could be a very good add.
lets me explain:

if i want to add many states classes this is how i'm supposed to do
```html
<div class="hover:border hover:bg-green-300 hover:border-green-500"></div>
```

My suggestion is to make the tailwind processor able to understand cascading states classes for example
```html
<div class="hover:border,bg-green-300,border-green-500"></div>
```
or
```html
<div class="hover:border|bg-green-300|border-green-500"></div>
```

in fact, the cascading action will use a symbol that's unique

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)

**Reply by su-angel** [reply]
ok thanks
there are talking about grouping, that's why my search on the topic were null; i used **cascading** as key word
